src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+.form-search {
+  margin-top: 20px;
+}
+
+.input {
+  display: block;
+  width: 100%;
+  color: rgb(61, 78, 97);
+  border: 1px solid rgb(225, 231, 237);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  outline: none;
+  font-family: inherit;
+  font-weight: 900;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.input-search-big {
+  height: 70px;
+  box-sizing: border-box;
+  padding-top: 25px;
+  padding-bottom: 26px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 26px 25px;
+}
+
+.input-search-big::placeholder {
+  font-family: inherit;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: rgb(61, 78, 97);
+  text-shadow: none;
+}
+
+.input-search-small {
+  height: 42px;
+  box-sizing: border-box;
+  padding-top: 11px;
+  padding-bottom: 11px;
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 11px;
+  background-position: 13px 15px;
```

Comment:
don't repeat yourself

refactor your code according to checklist point
<img width="764" alt="image" src="https://user-images.githubusercontent.com/102411130/224945634-f19b5d4f-15ea-4173-81ca-38eebfffbdde.png">